src/ontology/edits/oeo-physical.omn

```diff
@@ -3377,6 +3380,21 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/735",
         OEO_00000533 some OEO_00000007
     
     
+Class: OEO_00010084
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A reservoir is an artificial that stores liquid water and has a dam as part.",
+        <http://purl.obolibrary.org/obo/IAO_0000116> "Reservoirs created by dams provide water for activities such as hydro energy transformation, irrigation, human consumption, industrial use, aquaculture, and navigability. They are also used to suppress floods.",
```

Comment:
> They are also used to suppress floods.

I would prefer to use the word regulate instead of suppress. Maybe: _They are also used for flood regulation._ But I don't want to start a big discussion... if you don't like my suggestion, then go with your definition.

Reply:
I am fine with your proposal. The original formulation was simply an adaptation from Wikipedia.